packages/data-stores/src/queries/use-launchpad-navigator.ts

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
+
+export interface APIFetchOptions {
+	global: boolean;
+	path: string;
+}
+
+interface LaunchpadNavigatorResponse {
+	available_checklists: string[];
+	current_checklist: string;
```

Comment:
We discussed this synchronously: we should allow this value to be nullable.
```suggestion
	current_checklist: string | null;
```